charts/n8n/Chart.yaml

```
@@ -34,5 +34,8 @@ annotations:
   artifacthub.io/prerelease: "false"
   # supported kinds are added, changed, deprecated, removed, fixed and security.
   artifacthub.io/changes: |
-    - kind: fixed
-      description: "Adds namespace to all namespace-scoped templates: fixes https://github.com/8gears/n8n-helm-chart/issues/224"
+    - kind: added
+      description: Add support spec.containers.resizePolicy for all containers. This allows to control the behavior when a container is resized.
+      links:
+        - name: GitHub PR
+          url: https://github.com/8gears/n8n-helm-chart/pull/232
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)